kb/testing/understanding-ci-failures.md

```diff
@@ -0,0 +1,190 @@
+# Mbed TLS CI failure FAQ
+
+This guide is intended for contributors to Mbed TLS. It provides some tips to on investigating and resolving test failures in your contribution. It assumes basic familiarity with the CI jobs; please at least skim the [Mbed TLS CI guide](testing-ci.md).
+
+## Understanding PR test failures
+
+### PR tests failed. Which components failed?
+
+If the PR tests fail, the status on GitHub is set to a string that lists failed components. Note that due to a GitHub limitation, the list of failed components may be truncated. Click the “Details” link to see the full job breakdown.
+
+The default view on Jenkins unfortunately truncates job names. For more convenient access to the names and logs of failed components:
+
+1. Install the GreaseMonkey/TamperMonkey user script [JenkinsPipelineSteps](https://openuserjs.org/scripts/magnushakansson/JenkinsPipelineSteps).
+2. Go to the Jenkins “classic” view by clicking the ![Go to classic](jenkins-blueocean-go-to-classic.svg) icon near the top right, then follow the “Pipeline Steps” link in the left column.
```

Comment:
I don't understand why the SVG icon isn't rendering in the document.

Reply:
[According to W3C](https://www.w3.org/Graphics/SVG/IG/resources/svgprimer.html#SVG_in_HTML):

> At the current time, of the five browsers, only Opera, Safari and Chrome support svg as an image format in HTML

Changing it to a PNG seems to work, I'll raise that as a PR.

Reply:
Hmm, though actually that document was a long time ago, so it's probably out of date.

Reply:
https://caniuse.com/svg-img